1. Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800]. Найдите ее среднее значение и дисперсию.

$a = 200$

$b = 800$

$M(A) = \frac{a + b}{2}$

In [1]:
a = 200
b = 800
M = (a + b) / 2
M

500.0

$D(A) = \frac{(b - a)^2}{12}$

In [2]:
D = (b - a)**2 / 12
D

30000.0

2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.

$D(B) = 0.2$

$a = 0.5$

$D(B) = \frac{(b-a)^2}{12}$

$0.2 = \frac{(b-0.5)^2}{12}$

$b = \sqrt{12 \cdot 0.2}+0.5$ и $b = -\sqrt{12 \cdot 0.2}+0.5$

In [5]:
b1 = (12 * 0.2)**0.5 + 0.5
b1

2.049193338482967

In [6]:
b2 = -(12 * 0.2)**0.5 + 0.5
b2

-1.0491933384829668

Так как $b$ - правая граница, то $b  \thickapprox 2.05$

$M(B) = \frac{a + b}{2}$

In [7]:
a = 0.5
b = b1
M = (a + b) / 2
M

1.2745966692414834

3. Непрерывная случайная величина X распределена нормально и задана плотностью распределения

$f(x) = \frac{1}{4 \sqrt{2\pi}} e^{-\frac{(x+2)^2}{32}}$

Найдите:

а) $M(X)$

б) $D(X)$

в) $std(X)$ (среднее квадратичное отклонение)

$f(x) = \frac{1}{\sigma \sqrt{2 \pi}} e ^ {- \frac{(x - a)^2}{2 \sigma^2}},$

где $a = M(X), \; \sigma^2 = D(X)$

Таким образом, для исходной случайной величины $X$:

а) $M(X) = a = -2$

б) $D(X) = \sigma^2 = 16$

в) $std(X) = \sigma = 4$

4. Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:

а) больше 182 см

б) больше 190 см

в) от 166 см до 190 см

г) от 166 см до 182 см

д) от 158 см до 190 см

е) не выше 150 см или не ниже 190 см

ё) не выше 150 см или не ниже 198 см

ж) ниже 166 см

$a = 174$

$\sigma = 8$

$f(x) = \frac{1}{8 \sqrt{2 \pi}} e ^ {- \frac{(x - 174)^2}{128}}$

а) $x = 182 = 174+8 = a + \sigma$

$z = \frac{182-174}{8} = 1$

Из $z$-таблицы вероятность того, что случайная величина меньше $z$:

$0.8413$

Искомая вероятность:

$P = 1 - 0.8413 = 0.1587$

б) $x = 190 = 174+2 \cdot 8 = a + 2\sigma$

$z = \frac{190-174}{8} = 2$

Из $z$-таблицы вероятность того, что случайная величина меньше $z$:

$0.9772$

Искомая вероятность:

$P = 1 - 0.9772 = 0.0228$

в) $x_1 = 166 = 174-8 = a - \sigma$

$x_2 = 190 = 174 + 2 \cdot 8 = a + 2\sigma$

$z_1 = \frac{166-174}{8} = -1$

$z_2 = \frac{190-174}{8} = 2$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_1$:

$0.1587$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_2$:

$0.9772$

Искомая вероятность:

$P = 0.9772 - 0.1587 = 0.8185$

г) $x_1 = 166 = 174-8 = a - \sigma$

$x_2 = 182 = 174+8 = a + \sigma$

По правилу трех сигм искомая вероятность:

$P = 0.6826$

$z_1 = \frac{166-174}{8} = -1$

$z_2 = \frac{182-174}{8} = 1$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_1$:

$0.1587$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_2$:

$0.8413$

Искомая вероятность:

$P = 0.8413 - 0.1587 = 0.6826$

д) $x_1 = 158 = 174-2 \cdot 8 = a - 2\sigma$

$x_2 = 190 = 174+2 \cdot 8 = a + 2\sigma$

По правилу трех сигм искомая вероятность:

$P = 0.9544$

$z_1 = \frac{158-174}{8} = -2$

$z_2 = \frac{190-174}{8} = 2$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_1$:

$0.0228$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_2$:

$0.9772$

Искомая вероятность:

$P = 0.9772 - 0.0228 = 0.9544$

е) $x_1 = 150 = 174-3 \cdot 8 = a - 3\sigma$

$x_2 = 190 = 174+2 \cdot 8 = a + 2\sigma$

$z_1 = \frac{150-174}{8} = -3$

$z_2 = \frac{190-174}{8} = 2$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_1$:

$0.0014$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_2$:

$0.9772$

Искомая вероятность:

$P = 0.0014 + (1 - 0.9772) = 0.0242$

ё) $x_1 = 150 = 174-3 \cdot 8 = a - 3\sigma$

$x_2 = 198 = 174+3 \cdot 8 = a + 3\sigma$

По правилу трех сигм искомая вероятность:

$P = 1 - 0.9972 = 0.0028$

$z_1 = \frac{150-174}{8} = -3$

$z_2 = \frac{198-174}{8} = 3$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_1$:

$0.0014$

Из $z$-таблицы вероятность того, что случайная величина меньше $z_2$:

$0.9986$

Искомая вероятность:

$P = 0.0014 + (1 - 0.9986) = 0.0028$

ж) $x = 166 = 174-8 = a - \sigma$

$z = \frac{166-174}{8} = -1$

Из $z$-таблицы вероятность того, что случайная величина меньше $z$:

$0.1587$

Искомая вероятность:

$P = 0.1587$

5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

$z = \frac{x-M(X)}{\sigma} = \frac{x-M(X)}{\sqrt{D(X)}}$

In [1]:
x = 190
z = (x - 178) / 25**0.5
z

2.4

Таким образом, рост человека равный 190 см отклоняется от математического ожидания на $2.4 \sigma$.